packages/aws-cdk/lib/cli/cli-config.ts

```diff
@@ -158,6 +158,18 @@ export async function makeConfig(): Promise<CliConfig> {
               'and falls back to a full deployment if that is not possible. ' +
               'Do not use this in production environments',
           },
+          'hotswap-ecs-minimum-healthy-percent': {
+            type: 'string',
+            desc: 'Lower limit on the number of your service\'s tasks that must remain in the RUNNING state during a deployment, as a percentage of the desiredCount',
+          },
+          'hotswap-ecs-maximum-healthy-percent': {
+            type: 'string',
+            desc: 'Upper limit on the number of your service\'s tasks that are allowed in the RUNNING or PENDING state during a deployment, as a percentage of the desiredCount',
+          },
```

Comment:
These were just flat out missing from the yargs definition, even though we did (try to) handle them in:

https://github.com/aws/aws-cdk-cli/blob/382393730b6e8870cfb5d22aaf40c51b0b73b887/packages/aws-cdk/lib/cli/user-configuration.ts#L306-L310